README.md

```diff
@@ -94,6 +94,15 @@ For Windows, you can use the following commands:
 * `gradlew spotlessApply` an check and apply formatting to any plain-text file.</br>
 * `gradlew build`  provides a command line to execute build script.</br>
 
+### Instructions to get the latest apk
+
+To get the latest apk fom the Github actions artifacts, follow these steps
+
+1. Navigate to the [Actions](https://github.com/openMF/mobile-wallet/actions) tab of this repository.
+2. Click on your particular workflow from the workflows list.
```

Comment:
Umm, you know what, on following this one will get the apk generated for the lastest triggered workflow(that can be of any pull request which isn't merged in the dev) and the core idea of the corresponding issue is to make people able to download the latest stable apk, which would always be generated via latest workflow triggered on the `dev` branch. 

Let me know if some clarification is needed.

Reply:
Don't prioritize this, I'm not very sure of the idea. I'll leave it upto @devansh-299 regarding what apk we want people to download, the one corresponding to their Pull Request(which they can obtain by clicking the workflows from their PR directly) or the latest stable one.

Reply:
> Don't prioritize this, I'm not very sure of the idea. I'll leave it upto @devansh-299 regarding what apk we want people to download, the one corresponding to their Pull Request(which they can obtain by clicking the workflows from their PR directly) or the latest stable one.

@s-ayush2903 Sure! I'll do this if he says so. Also thanks a lot for the review.

Reply:
> 
> 
> Don't prioritize this, I'm not very sure of the idea. I'll leave it upto @devansh-299 regarding what apk we want people to download, the one corresponding to their Pull Request(which they can obtain by clicking the workflows from their PR directly) or the latest stable one.

The main idea of building APKs and saving them as artifacts was to allow people to test out the latest stable build without having the need of installing Android Studio on their systems. The contributors already have the environment set up to build their projects and test it, hence it makes sense to link the actions for merged changes.

Reply:
Okay, so @Sparsh1212, so update accordingly i.e., specifying workflow query to the `push` event to the `dev` branch